myapp/app/controllers/tasks_controller.rb

```diff
@@ -2,11 +2,19 @@
 
 class TasksController < ApplicationController
   before_action :set_task, only: %i[show edit update destroy]
+  before_action :logged_in_user
   def index
     @tasks = if Task.statuses.keys.include?(params[:status])
-               Task.includes([:user]).where(status: params[:status]).task_name_partial_search(params[:task_name]).page(params[:page])
+                Task.includes([:user])
+                  .user_id_search(current_user.id)
+                  .task_name_partial_search(params[:task_name])
+                  .where(status: params[:status])
+                  .page(params[:page])
```

Comment:
このようにメソッドチェーンを改行するときは、ドットの開始位置をあわせましょう
```suggestion
                Task.includes([:user])
                    .user_id_search(current_user.id)
                    .task_name_partial_search(params[:task_name])
                    .where(status: params[:status])
                    .page(params[:page])
```